packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -46,37 +47,44 @@ function registerDebugHandlers(): vscode.Disposable {
       if (event && event.session && event.session.type === DEBUGGER_TYPE) {
         if (event.event === GET_LINE_BREAKPOINT_INFO_EVENT) {
           console.log('in registerDebugHandlers, getting line breakpoint info');
-          event.session.customRequest(
-            LINE_BREAKPOINT_INFO_REQUEST,
-            breakpointUtil.getRawLineBPInfo()
+          const sfdxApex = vscode.extensions.getExtension(
+            'salesforce.salesforcedx-vscode-apex'
           );
+          if (sfdxApex && sfdxApex.exports) {
+            const lineBpInfo = await sfdxApex.exports.getLineBreakpointInfo();
+            event.session.customRequest(
+              LINE_BREAKPOINT_INFO_REQUEST,
+              lineBpInfo
+            );
+            console.log(
+              'in registerDebugHandlers, retrieved line breakpoint info from language server'
+            );
+          }
         }
       }
     }
   );
-  return vscode.Disposable.from(customEventHandler);
-}
 
-export async function activate(context: vscode.ExtensionContext) {
-  console.log('Apex Replay Debugger Extension Activated');
-  const commands = registerCommands();
-  const debugHandlers = registerDebugHandlers();
-  const debugConfigProvider = vscode.debug.registerDebugConfigurationProvider(
-    'apex-replay',
-    new DebugConfigurationProvider()
+  const config = vscode.workspace.getConfiguration();
+  const checkpointsEnabled = config.get(
```

Comment:
You already do this in `activate()`, just pass it in to this method instead of doing it again.

Reply:
done